unit1-wrap-up/activity-mind-map.md

```diff
@@ -0,0 +1,13 @@
+# Unit 1 - Reflection Activity
+
+Wow! We've learned so much in the last 5 weeks. Let's take time to reflect on our learning and make connections.
+
+We will use a technique called a mind map. A mind map is a diagram that can represent tasks, ideas, concepts, etc. and the connections between them. In our case, we can include anything from our learnings from Unit 1 -- whether it be python programming concepts, study skills, learnings about how to be in community, or anything else.
+
+These are a few tools our team recommends: 
+* [Google Slide](https://www.google.com/slides/about/)
+* [Mind Meister](https://www.mindmeister.com/)
+
+and here is a [more exhaustive list](https://zapier.com/blog/best-mind-mapping-software/) of mind mapping software.
+
+We encourage you to pick a tool and go for it -- no censorship needed.
```

Comment:
Recommending for this be reworded so the "we" is about us learners, without the instructor/team divide.

Also recommending "we will" -> "let's" since it's about to happen.

- Capitalization suggestions
- Framing the directions as activity instructions instead of as suggestions
```suggestion
Let's create a mind map. A mind map is a diagram that can represent tasks, ideas, and concepts, and the connections between them.

In our case, let's include any topic from our learnings from this unit. We should consider:

- Python programming concepts
- Study skills
- Learnings about how to be in community
- Anything else!

We can make our mind map using any of these tools:

- [Google Slide](https://www.google.com/slides/about/)
- [Mind Meister](https://www.mindmeister.com/)
- [Software from this list of mind mapping software](https://zapier.com/blog/best-mind-mapping-software/)
```

Reply:
I didn't see this before I started making changes, so I incorporated it, but didn't straight up commit the suggestion.